framework/tlvf/yaml/tlvf/ieee_1905_1/tlvWscM2.yaml

```diff
@@ -76,6 +76,5 @@ tlvWscM2:
   configuration_error_attr: WSC::sWscAttrConfigurationError
   device_password_id_attr: WSC::sWscAttrDevicePasswordID
   os_version_attr: WSC::sWscAttrOsVersion
-  vendor_extensions_attr: WSC::sWscAttrVendorExtension
```

Comment:
The Version2 attribute must be present here, according to the WSC spec. The BBF code doesn't check for it, but maybe other implementations do.

BTW, in prplMesh-old, I added the WFA extension directly in this vendor extension (so there was one vendor extension 2 IEs: version2 and MultiAP). I had missed the part specifying that it was to be in the encrypted section.

Reply:
marking as resolved, this commit is removed and the PR is rebased on top of #246